demisto_sdk/commands/common/tools.py

```diff
@@ -1107,6 +1107,27 @@ def old_get_latest_release_notes_text(rn_path):
     return new_rn if new_rn else None
 
 
+def find_pack_folder(path: Path) -> Path:
```

Comment:
There is something similar named get_pack_name but it return the PACKS index in the path not the path, more over I added it because @mmhw works on PR #3354 and will use it.